prac_05/wimbledon.py

```diff
@@ -0,0 +1,51 @@
+"""
+Game, Set, Match
+Estimate: 60 min
+Actual:   66 min
+"""
+
+FILENAME = "wimbledon.csv"
+
+
+def main():
+    """Read and process data file, then display data"""
+    pairs = get_data(FILENAME)
+    countries, champion_to_win_count = process_data(pairs)
```

Comment:
Hi. Congrats on getting this to work! It was a hard question. 
The techniques you used looks good to me. 

This is a minor suggestion and you don't have to change your code (unless you want to). Do you think `process_data()` does just one logical task (SRP)? In some ways it does, but maybe it could be broken down into functions, something like extract_countries(data) and extract_champion_wins(data) to make the code easier to read and test. It may be a little less efficient because you have to do the for loop twice, but feels better for SRP and would be easier to read.